docs/url/url.md

```diff
@@ -25,11 +25,11 @@ This document defines semantic conventions that describe URL and its components.
 <!-- semconv url -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
```

Comment:
I wonder now with the registry, if we even need this page? Seems to be just duplicating things?

Reply:
Yes, particularly for this small attribute it's seems to be identical, but not for all attributes. Also I would need to update registry `url.md` with sensitive information from this file
This means that this file will be also removed from https://opentelemetry.io/docs/specs/semconv/ page, is it ok?

Reply:
Yeah. Let's leave it for now, we can revisit them later, once the registry is complete.